libr/core/cconfig.c

```diff
@@ -538,10 +604,9 @@ static int cb_emustr(void *user, void *data) {
 static int cb_emuskip(void *user, void *data) {
 	RConfigNode *node = (RConfigNode*) data;
 	if (*node->value == '?') {
-		r_cons_printf ("Concatenation of meta types encoded as characters:\n" \
-				"'d': data\n'c': code\n's': string\n'f': format\n'm': magic\n" \
-				"'h': hide\n'C': comment\n'r': run\n" \
-				"(default is 'ds' to skip data and strings)\n");
+		print_node_options (node);
+		r_cons_printf ("Concatenation of meta types encoded as characters.\n" \
```

Comment:
or put it with eprintf. but it looks noisy to me.or maybe you can add this with ??